tt1-blocks/assets/css/blocks.css

```diff
@@ -1,8 +1,8 @@
 /*--------------------------------------------------------------
-# Cover
+# Border, for backwards compatibility
 --------------------------------------------------------------*/
 
-.wp-block-cover.is-style-twentytwentyone-border {
+.is-style-twentytwentyone-border {
```

Comment:
this is needed so we don't lose the border on already inserted patterns with the old markup. I hope by the end of this PR we can move this to style.css